app/views/guests/show.html.erb

```diff
@@ -0,0 +1,19 @@
+<h1><%= @guest.name %></h1>
+
+<h2>Rooms Stayed In:</h2>
+<ul>
+  <% @guest.rooms.each do |room| %>
+    <li>
+      <p><strong>Hotel: <%= room.hotel.name %>,</strong><p>
+      Suite: <%= room.suite %>,
+      Rate: $<%= room.rate %>
+    </li>
+  <% end %>
+</ul>
+
+<h3>Add a Stay for this Guest:</h3>
+<%= form_with url: "/guests/#{@guest.id}/add_room", method: :post do |form| %>
```

Comment:
A more conventional RESTful path here would be "/guests/:guest_id/rooms" and the fact that it's a POST sort of indicates that you're creating the association between the room and the guest. It's generally more common with REST to have the method (POST, in this case) dictate the action, and then have the path indicate the resources.